cmd/root.go

```diff
@@ -58,27 +73,34 @@ func Execute() {
 }
 
 func init() {
-	// cobra.OnInitialize(initConfig)
-
-	// RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.namerctl.yaml)")
-
-	RootCmd.PersistentFlags().StringVar(&baseURLString, "base-url",
-		os.Getenv("NAMERCTL_BASE_URL"),
+	//jww.SetStdoutThreshold(jww.LevelTrace)
```

Comment:
I chose to leave these in so that it's easy to enable debug logging for viper during development. It's sort of hard to remember that you need the "jwalterweatherman" library...